microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -159,218 +235,327 @@ If you publish a person's Personal Information to an outside service or third-pa
 
 
 <!-- ====================================================================== -->
-## Edge Add-ons Certification Complaint and Appeal Process
-
-All the extensions should adhere to the store policies listed above. If your extension failed in the review process, please review the store policies to understand the reason for failure. After submitting your extension using Partner Center, to ask a question about the review or certification status of it, go to [New Support Request](https://support.microsoft.com/supportrequestform/e7a381be-9c9a-fafb-ed76-262bc93fd9e4) and complete the form.
+## Complaint and appeal process for Microsoft Edge Add-ons certification 
 
-### Microsoft Edge Add-ons appeal statistics for FY2022
+All extensions must adhere to the store policies listed above.  If your extension failed in the review process, review the store policies to understand the reason for failure.  After submitting your extension using Partner Center, to ask a question about the review or certification status of it, go to [New Support Request](https://support.microsoft.com/supportrequestform/e7a381be-9c9a-fafb-ed76-262bc93fd9e4) and complete the form.
 
-Numbers as reported on 7/1/2023
 
+<!-- ------------------------------ -->
+#### Appeal statistics for Microsoft Edge Add-ons for FY2022
 
 | Statistic                                                             | Count     |
 |-----------------------------------------------------------------------|-----------|
-| App and/or Account Enforcement Action Appeals                         |        34 |
-| Complaints about Technological Issues                                 |         0 |
-| Regulatory Compliance Complaints                                      |         0 |
+| App and/or account enforcement action appeals                         |        34 |
+| Complaints about technological issues                                 |         0 |
+| Regulatory compliance complaints                                      |         0 |
 | Questions about certification, policy, submission, and technical help |       131 |
 | Miscellaneous                                                         |         5 |
-| Total Issues                                                          |       175 |
+| Total issues                                                          |       175 |
 | Overturned decisions                                                  |        28 |
-| Average Processing Time                                               | 6.58 days |
+| Average processing time                                               | 6.58 days |
+
+These numbers were reported on July 1, 2023|January 7, 2023.<!-- todo: which? -->
```

Comment:
@Bhuvana-Priya - were these numbers reported on July 1, or January 7?

Reply:
This appeal stats : July 1

Reply:
These numbers were reported on July 1, 2023.